addons/frag/functions/fnc_addBlacklist.sqf

```diff
@@ -0,0 +1,36 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Jaynus, NouberNou, Lambda.Tiger
+ * Adds a round to the blacklist (will be ignored) and removes any ace_frag event handlers added to it.
+ * Once blacklisted a projectile can not be "unblacklisted."
+ *
+ * Arguments:
+ * 0: Projectile to be blacklisted <OBJECT>
+ *
+ * Return Value:
+ * Was the projectile blacklisted <BOOL>
+ *
+ * Example:
+ * [_projectile] call ace_frag_fnc_addBlackList
+ *
+ * Public: Yes
+ */
+
+params [["_projectile", objNull, [objNull]]];
+TRACE_2("addBlackList",_projectile,typeOf projectile);
+
+if (isNull _projectile || {!isClass (configFile >> "CfgAmmo" > (typeOf _projectile))}) exitWith {
```

Comment:
Bad syntax:
```suggestion
if (isNull _projectile || {!isClass (configFile >> "CfgAmmo" >> typeOf _projectile)}) exitWith {
```

Reply:
Gosh, I can't believe I missed that. I really appreciate it.